baybe/utils/plotting.py

```diff
@@ -0,0 +1,64 @@
+"""Utilities that are useful for plotting."""
+
+import json
+import os
+import sys
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def create_plots(data: pd.DataFrame, name: str, **kwargs) -> None:
+    """Create plots from a given data frame and save them as a svg file.
+
+    The plots will be saved in the location in which the file that calls this function
```

Comment:
first sentence would become redundant

Reply:
Not quite - due to having both `path` and `base_name` (see my other comment), this is no longer redundant (at least in my opinion).